js/index.js

```diff
@@ -379,13 +385,46 @@ webix.ready(function(){
     },
     url:products_data_link
   };
-
+  var admin_categories = {
+    view:"datatable",
+    id:"admin_categories",
+    scrollX:false,
+    select:true,
+    editable:true,
+    editaction:"dblclick",
+    columns:[
+      { id:"value",	header:"Category", fillspace:true, editor:"text"}
+    ],
+    rules:{
+      value:function(val){
+        return  titleValidation(val,20);
+      }
+    }
+  };
+  var admin_categories_toolbar = {
+    view:"toolbar",
+    cols:[
+      { view:"button", value:"Add new", css:"webix_primary",
+        click:function(){
+          categories_data_collection.add({value:"New category"});
```

Comment:
there should be a `.showItem()` call or the item should be added in the first position, otherwise it will be not visible in case of long data

Reply:
I have used my custom function for scrolling and selecting new elements: scrollToLastAddedElement(admin_categories,true,true);